modules/ROOT/pages/common-create-keystore-task.adoc

```diff
@@ -1,7 +1,7 @@
-= To Create a Java KeyStore and Trust Certificates
+= Create a Java KeyStore and Trust Certificates
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
 [NOTE]
-Information in this topic has moved to xref:tls-configuration.adoc[Configure TLS with Keystores and Truststores]. Please update your bookmarks as this file will be removed after January 1, 2018.
\ No newline at end of file
+Information in this topic has moved to xref:tls-configuration.adoc[Configure TLS with Keystores and Truststores]. Please update your bookmarks as this file will be removed after January 1, 2018.
```

Comment:
Does this Note still need to be here since the file has already been removed and they won't be able to bookmark it?